tools/scripts/make_pl_poke_icon.py

```diff
@@ -0,0 +1,594 @@
+#!/usr/bin/env python3
```

Comment:
I started writing this in meson originally, but it got unwieldy. I decided that it was a good fit for a python sub-script.